docs/user-guides/cmake-user.rst

```diff
@@ -5,19 +5,20 @@ CMake user
 ----------
 
 This kind of developer can **read** CMake code that was written by more
-experienced CMake developer. He understand some simple features like that
-executable added by ``add_executable`` command and this command contains list
-of sources files.  But probably will not be able to tell why in some cases
-``include_directories`` command called but ``target_include_directories`` in
-other cases. The main target of modifications is C++ code.
+experienced CMake developer.  They understand some simple features such as
```

Comment:
double space between sentence, should be removed

Reply:
Thank you for catching that.